metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -222,19 +229,11 @@ final class BloopServers(
     ) match {
       case Failure(exception) => Future.failed(exception)
       case Success(_) =>
-        languageClient
-          .showMessageRequest(
-            Messages.BloopJvmPropertiesChange.params(bloopJsonUpdateCause)
-          )
-          .asScala
-          .flatMap {
-            case messageActionItem
-                if messageActionItem == Messages.BloopJvmPropertiesChange.reconnect =>
-              shutdownServer()
-              reconnect().ignoreValue
-            case _ =>
-              Future.unit
-          }
+        scribe.info(
```

Comment:
I decided that we don't need to ask in case the json file is managed by Bloop, since that will confuse new users. Especially that we set the javaHome at the start.

Reply:
It seems to me that when the restart is done due to update to Metals Java home, the users need a way to know this is going to affect their Bloop java home too. Don’t they?

Reply:
The main problem is that we will also do this at the start and we would always show it to the user even if it's a new user that doesn't know anything about the Java home etc. So I decide that it's better to hide it currently instead. Otherwise I expect a lot of questions from confused users.